.github/workflows/documentation.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     name: TF docs
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4
```

Comment:
This is overwritting another change. We need to not have any update in your PR here. Please ensure this is not changed in your PR.